src/app/submit-item/components/submit/submit.component.html

```diff
@@ -134,12 +133,22 @@ <h4>Wie möchtest du dein Ergebnis erhalten?</h4>
 
 <ng-template #finish>
   <div class="page-container-column">
-    <h1>Danke, <span class="alt">dass du deinen Fall eingereicht hast!</span></h1>
+    <h1 *ngIf="!mailGiven">Danke, <span class="alt">dass du deinen Fall eingereicht hast!</span></h1>
+    <h1 *ngIf="mailGiven">Nur noch ein Schritt - <span class="alt"> Mailadresse bestätigen</span></h1>
```

Comment:
You could wrap the heading and paragraph in one single div and check the mailGiven property only once.

Reply:
But then we would have to duplicate the other components (button group and image), because the heading and the content are two separate div's. I think that would inflate the code more than having two checks.

Reply:
You're right 👍